src/php/Analyzer.php

```diff
@@ -87,10 +85,7 @@ private function isLiteral($x): bool {
     }
 
     /**
-     * @param Phel|scalar|null $x
-     * @param ?NodeEnvironment $nodeEnvironment
-     * 
-     * @return Node
+     * @param mixed $x
```

Comment:
`mixed` is much more open than `Phel|scalar|null`. The original value was correct.